src/OrbitGl/TimelineUi.h

```diff
@@ -11,6 +11,13 @@
 
 namespace orbit_gl {
 
+namespace {
+
+const Color kMajorTickColor(255, 254, 253, 255);
+const Color kMinorTickColor(255, 254, 253, 63);
```

Comment:
This is now in the code base usable as `orbit_gl::kMajorTickColor`, right? Maybe then include the word timeline: `kTimelineMajorTick`. Or make it part of the TimelineUi class: `orbit_gl::TimelineUi::kTimelineMajorTick`